src/Pie.js

```diff
@@ -41,14 +41,58 @@ const RingShape = props =>
   Platform.OS === 'ios'
     ? <ArcShape {...props} startAngle={0} arcAngle={360} />
     : <Group>
-        <ArcShape {...props} startAngle={0} arcAngle={180} />
-        <ArcShape {...props} startAngle={180} arcAngle={180} />
-      </Group>
+      <ArcShape {...props} startAngle={0} arcAngle={180} />
+      <ArcShape {...props} startAngle={180} arcAngle={180} />
+    </Group>
+
+const Dividers = ({ dividers, dividerColorOverlays, strokeCap }) => {
```

Comment:
I'm trying to work out a different solution to accomplish this. I did it this way to avoid double iteration over the `sections` array.

The colorOverlay is required for round strokeCap